src/plays/react-todo-app/ReactTodoApp.jsx

```diff
@@ -11,50 +11,81 @@ import './react-todo.css';
 import Input from './components/Input';
 import TodoList from './components/TodoList';
 
+function getLocalStorageTodos() {
+  const todos = localStorage.getItem('todos');
```

Comment:
Always use keys that are unique. May be another play have same key name and override yours.

Reply:
> Always use keys that are unique. May be another play have same key name and override yours.

Added folder name before localstorage key to make it more unique - (eg: useLocalStorage('react-todo-app/todos', []), useLocalStorage('keeper/notes', []))